Sources/Build/BuildDelegate.swift

```diff
@@ -570,6 +570,14 @@ fileprivate struct CommandTaskTracker {
             totalCount -= 1
             break
         case .isComplete:
+            if (totalCount == finishedCount) {
+                let latestOutput: String? = latestFinishedText
+                latestFinishedText = """
+                    \(latestOutput ?? "")
+                    * Build Process Completed!
+                    * Generated Binaries: .build/
```

Comment:
This should probably not be hardcoded.  There should be a way to get this still from the underlying code.  I don't know offhand how to do that but can take a look.  You should be able to find it by searching for this string in the code.